src/main/java/seedu/duke/command/CommandListClientsWithTags.java

```diff
@@ -34,43 +42,51 @@ public void execute(Ui ui, Storage storage, PropertyList propertyList, ClientLis
             displayClientShort(clientList, ui);
             break;
         default:
+            //no errors are possible at this stage. Any potential errors are handled earlier
             break;
         }
+        //newline at the end of command
+        ui.printNewline();
     }
 
     private void displayClientContacts(ClientList clientList, Ui ui) {
-        for (int i = 0; i < clientList.getCurrentListSize(); i++) {
+        for (int i = CommandStructure.START_INDEX; i < clientList.getCurrentListSize(); i++) {
             ui.displayOneClientContact(clientList.getClientList().get(i), i + 1);
         }
-        ui.displayNoOfClients(clientList.getCurrentListSize());
+        ui.displayNoOfClients(clientList.getCurrentListSize(),
+                clientList.getCurrentListSize() == CommandStructure.ONE_ITEM_IN_LIST);
```

Comment:
You might want to create a boolean variable to store `clientList.getCurrentListSize() == CommandStructure.ONE_ITEM_IN_LIST` before passing that variable into the method. This makes it easier to read also.